projects/ngrx.io/content/examples/store/src/app/counter.actions.ts

```diff
@@ -1,20 +1,6 @@
 // #docregion
-import { Action } from '@ngrx/store';
+import { createAction } from '@ngrx/store';
 
-export enum ActionTypes {
-  Increment = '[Counter Component] Increment',
-  Decrement = '[Counter Component] Decrement',
-  Reset = '[Counter Component] Reset',
-}
-
-export class Increment implements Action {
-  readonly type = ActionTypes.Increment;
-}
-
-export class Decrement implements Action {
-  readonly type = ActionTypes.Decrement;
-}
-
-export class Reset implements Action {
-  readonly type = ActionTypes.Reset;
-}
+export const increment = createAction('[Counter Component] Increment');
```

Comment:
Just to be sure, I think we're keeping this as simple as possible.
But here we're not using a `map` to define actions, nor the `union`.

Reply:
Same as above